app/helpers/check_answers_helper.rb

```diff
@@ -1,161 +1,218 @@
-      next link_to_parent_page(question, "construction_services") if question.eql?("construction_cost")
-      next link_to_parent_page(question, "it_services") if question.eql?("it_services_other")
-      next link_to_parent_page(question, "it_services") if question.eql?("it_cost")
-      next link_to_parent_page(question, "transport_type") if question.eql?("transport_cost")
-      next link_to_parent_page(question, "offer_care_qualifications") if question.eql?("care_cost")
-      next link_to_parent_page(question, "rooms_number") if question.eql?("accommodation_cost")
-      next link_to_parent_page(question, "offer_space_type") if question.eql?("space_cost")
+  def staff_numbers
+    {
+      cleaners_number: session[:offer_staff_number][:cleaners_number],
+      developers_number: session[:offer_staff_number][:developers_number],
+      medical_staff_number: session[:offer_staff_number][:medical_staff_number],
+      office_staff_number: session[:offer_staff_number][:office_staff_number],
+      security_staff_number: session[:offer_staff_number][:security_staff_number],
+      trainers_or_coaches_number: session[:offer_staff_number][:trainers_number],
+      translators_number: session[:offer_staff_number][:translators_number],
+      other_staff_number: session[:offer_staff_number][:other_staff_number],
+    }.with_indifferent_access
+  end
 
-      value = concat_answer(session[question], question)
+  def space_numbers
+    {
+      other_number: session[:offer_space_type_other],
+      warehouse_space_number: session[:warehouse_space_description],
+      office_space_number: session[:office_space_description],
+    }.with_indifferent_access
+  end
 
+  def question_items(question)
+    if session[question].is_a?(Array)
       [{
-        field: t("coronavirus_form.questions.#{question}.title"),
-        value: sanitize(value),
-        edit: {
-          href: "#{question.dasherize}?change-answer",
-        },
+        field: I18n.t("coronavirus_form.questions.#{question}.title"),
+        value: session[question],
+        type: :list,
+      }]
+    else
+      [{
+        field: I18n.t("coronavirus_form.questions.#{question}.title"),
+        value: session[question],
       }]
     end
-
-    items.compact.flatten
   end
 
-  def skip_question?(question)
-    return true if question.in?(EXCLUDED_QUESTIONS)
-
-    question.in?(SKIPPABLE_QUESTIONS) && session[question].blank?
+  def accommodation_items
+    [
+      {
+        field: I18n.t("coronavirus_form.check_your_answers.sections.accommodation.type"),
+        value: session[:rooms_number],
+      },
+      {
+        field: I18n.t("coronavirus_form.check_your_answers.sections.accommodation.description"),
+        value: session[:accommodation_description],
+      },
+      {
+        field: I18n.t("coronavirus_form.check_your_answers.charge"),
+        value: session[:accommodation_cost],
+      },
+    ]
```

Comment:
I think an I18n scope would be useful here to reduce the length of some of the translations paths. Also, `I18n.t("coronavirus_form.check_your_answers.charge")` is used repeatedly so could be defined separately. So for example, if these two methods were added to the file:
```ruby
def sections_scope
  "coronavirus_form.check_your_answers.sections"
end

def charge_text
  I18n.t("coronavirus_form.check_your_answers.charge")
end
```
Then this method could be rewritten as:
```ruby
def accommodation_items
    [
      {
        field: I18n.t("accommodation.type", scope: sections_scope),
        value: session[:rooms_number],
      },
      {
        field: I18n.t("accommodation.description", scope: sections_scope),
        value: session[:accommodation_description],
      },
      {
        field: charge_text,
        value: session[:accommodation_cost],
      },
    ]
end
```
And the other method in this file could similarly be simplified.

Reply:
I didn't know you could use scope like that. Nice!

Reply:
Thanks Rob, we went with this suggestion 👍